ut-robomaster/src/subsystems/turret/turret_subsystem.cpp

```diff
@@ -27,6 +28,9 @@ TurretSubsystem::TurretSubsystem(src::Drivers* drivers)
 {
 }
 
+// should this be an AND??
+bool TurretSubsystem::hardwareOk() { return yaw.isOnline() || pitch.isOnline(); }
+
```

Comment:
Yeah can you change this to an AND? This looks like the reason that yaw was still running with one motor disconnected (because pitch was also working). I think it'd be worth ultimately splitting yaw and pitch into separate subsystems to help with this, but that can be a separate issue once this is merged.